src/Hotel.js

```diff
@@ -0,0 +1,43 @@
+import Room from "./Room";
+import Booking from "./Booking";
+
+class Hotel {
+  constructor(roomsData, bookingsData) {
+    this.allRooms = this.makeNewRooms(roomsData) || [];
+    this.allBookings = this.makeNewBookings(bookingsData) || [];
+  }
+
+  makeNewRooms(roomsData) {
+    return roomsData.map((room) => {
+      return new Room(room)
+    })
+  }
+
+  makeNewBookings(bookingsData) {
+    return bookingsData.map((booking) => {
+      return new Booking(booking)
+    })
+  }
+
```

Comment:
Similar to `customer repo`, right now instead `this.allRooms` and `this.allBookings` defaulting to `[]` it would throw an error on lines `11` and `17`.